CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 92.0.0
```

Comment:
This change should probably be release `91.1.0`, since there's no breaking or incompatible changes being introduced.

Also [we recommend](https://docs.publishing.service.gov.uk/manual/publishing-a-ruby-gem.html#versioning) that the version be bumped in a separate PR, to avoid any versioning conflicts if another release is made by someone else in the meantime.